.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
this file shouldn't be here.

Reply:
Jepp, I removed it. Thanks for catching that! 👍🏼